host/tests/rmfAudioClasses/rmfAudio.py

```diff
@@ -0,0 +1,528 @@
+                {
+                    "query_type": "direct",
+                    "query": "Enter file name and location to create output filename (example - /tmp/output.wav) :",
+                    "input": str(file_path)
+                }
+        ]
+
+        result = self.utMenu.select(self.testSuite, "Write output wav file", promptWithAnswers)
+
+    def startJitterTest(self, capture_type:int=1, threshold:int=16384, jitter_interval:int=100000, jitter_test_duration:int=120):
+        """
+        Starts jitter test with required parameters for test
+
+        Args:
+            capture_type (int, optional): 1 for primary data capture (default), 2 for auxiliary data capture.
+            threshold (int, optional): Threshold to check jitter against (default is 16k, 16384).
+            jitter_interval (int, optional): Interval in microseconds to monitor buffer for jitter (default is 100ms, 100000).
+            jitter_test_duration (int, optional): Test duration in seconds for jitter test (default is 120 seconds).
+
+        Returns:
+            None
+        """
+        promptWithAnswers = [
+                {
+                    "query_type": "direct",
+                    "query": "Select the audio capture type :",
+                    "input": str(capture_type)
+                },
+                {
+                    "query_type": "direct",
+                    "query": "Enter minimum threshold in bytes to check jitter : ",
+                    "input": str(threshold)
+                },
+                {
+                    "query_type": "direct",
+                    "query": "Enter interval in microseconds to monitor buffer for jitter : ",
+                    "input": str(jitter_interval)
+                },
+                {
+                    "query_type": "direct",
+                    "query": "Enter test duration in seconds for jitter test : ",
+                    "input": str(jitter_test_duration)
+                }
+        ]
+
+        result = self.utMenu.select(self.testSuite, "Start Jitter test", promptWithAnswers)
+
+    def checkJitterTestResult(self, capture_type:int=1):
+        """
+        Gets the result of jitter test
+
+        Args:
+            capture_type (int, optional): 1 for primary data capture (default), 2 for auxiliary data capture.
+
+        Returns:
+            bool : true/false based on jitter detected strings from C-test
+        """
+        promptWithAnswers = [
+                {
+                    "query_type": "direct",
+                    "query": "Select the audio capture type :",
+                    "input": str(capture_type)
+                }
+        ]
+
+        result = self.utMenu.select(self.testSuite, "Check jitter test result", promptWithAnswers)
+        jitter_pattern = "Jitter Detected !"
+
+        jitter_result = self.searchPattern(result, jitter_pattern)
+        if jitter_result:
+            return False
+        else:
+            return True

```

Comment:
need to a all get functions as menu items

Reply:
Updated existing menu item for RMF_AudioCapture_GetDefaultSettings, and added menu items in both c-test, python with new ones for RMF_AudioCapture_GetCurrentSettings and RMF_AudioCapture_GetStatus.. Updated testConfig yml to reflect same.